Azure-Samples/chat-with-your-data-solution-accelerator
# refactor: make backend helpers pep8 compliant

## Purpose

* Make backend helpers pep8 compliant.
* Resolves #855 

## Does this introduce a breaking change?



## How to Test

* Run the tests:
```
poetry run pytest -m "not azure and not functional"    
poetry run pytest -m "functional"  
```

* Run the application locally and go to http://localhost:8502/Ingest_Data
* Upload a document on the above endpoint
* Verify that the chunks are created


## What to Check

* Verify that the chunks are created
